api/server.spec.js

```diff
@@ -18,17 +18,17 @@ describe("server", () => {
         supertest(server)
           .get("/")
           // .expect(200) // from supertest
-          .then(response => {
+          .then((response) => {
             // from jest
-            expect(response.status).toBe(200);
+            expect(Array.isArray(response.body)).toBe(false);
           })
       );
     });
 
     it("should return { api: 'up' }", () => {
       return supertest(server)
         .get("/")
-        .then(response => {
+        .then((response) => {
```

Comment:
Is this prettier's handiwork or are you going back and adding parenthesis around the argument?